Survey/Sources/Presentation/Views/ViewModifier/TextViewPlaceholderModifier.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  TextViewPlaceholderModifier.swift
+//  Survey
+//
+//  Created by Khanh on 10/09/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import SwiftUI
+
+struct TextViewPlaceholderModifier: ViewModifier {
+
+    var showPlaceHolder: Bool
```

Comment:
```suggestion
    var showPlaceholder: Bool
```

for keeping consistency with
```
    @State var placeholder: String
```